sopel/modules/weather.py

```diff
@@ -1,14 +1,19 @@
 # coding=utf-8
-# Copyright 2008, Sean B. Palmer, inamidst.com
-# Copyright 2012, Elsie Powell, embolalia.com
-# Licensed under the Eiffel Forum License 2.
+"""
+weather.py - Sopel Yahoo! Weather Module
+Copyright 2008, Sean B. Palmer, inamidst.com
+Copyright 2012, Edward Powell, embolalia.net
+Licensed under the Eiffel Forum License 2.
+
+http://sopel.chat
+"""
```

Comment:
I'm guessing you hadn't pulled recently, because you're wiping out the change from ee86f38ddf07213d1dafb4939b71dea7a47809f3 here, I assume unintentionally.